README.md

```diff
@@ -62,6 +78,72 @@ python directory_setup.py
+
+<details>
+
+<summary><b>Command line options</b></summary>
+
+### Command line options
+
+Below is a short summary of what each command line option does. To view a more detailed description of what the run parameters do, see the [Documentation](https://impactlab.org/research/data-driven-spatial-climate-impact-model-user-manual-version-092023-epa/) for Data-driven Spatial Climate Impact Model (DSCIM). 
+
+#### Sector
+
+The user may only select one sector per run. Sectors represent the combined SC-GHG or partial SC-GHGs of the chosen sector.
+
+#### Discount rate
+
+These runs use endogenous Ramsey discounting that are targeted to begin at the chosen near-term discount rate(s). 
+
+#### Pulse years
+
+Pulse year represents the SC-GHG for a pulse of greenhouse gas (GHG) emitted in the chosen pulse year(s). 
+
+#### Domain of damages
+
+The default is a global SC-GHG accounting for global damages in response to a pulse of GHG. The user has the option to instead limit damages to those occurring directly within the territorial United States. This is only a partial accounting of the cost of climate change to U.S. citizens and residents because it excludes international transmission mechanisms, like trade, cross-border investment and migration, damage to the assets of U.S. citizens and residents outside the United States, or consideration of how GHG emission reduction activity within the United States impacts emissions in other countries.
+
+#### Optional files
+
+By default, the script will produce the expected SC-GHGs as a `.csv`. The user also has the option to save the full distribution of 10,000 SC-GHGs -- across emissions, socioeconomics, and climate uncertainty -- as a `.csv`, and the option to save global consumption net of baseline climate damages ("global_consumption_no_pulse") as a netcdf `.nc4` file.
+</details>
+
+<!-- Consider adding a section that suggests a test run of the default SC-GHGs. -->
+
+<!-- Consider splitting the next sections into their own markdown README file -->
+
+# DSCIM + FACTS Run process overview
+Ignoring environment setup for a moment, the general run process for use-case 2 -- running DSCIM-FACTS-EPA with exogenous climate inputs and using FACTS to generate GMSL from GMST and OHC files -- is summarized here. Detailed instructions for each step are provided later in the README.
+ 
+1. Format user GMST and OHC files manually as specified earlier (See [Formatting files](#formatting-files))
+<!-- Check this next one (#2) is still correct/necessary -->
+2. Place formatted GMST/OHC files into `dscim-facts-epa/scripts/input/climate`
```

Comment:
@JMGilbert is step 2. still accurate? the facts run script does this automatically I think so perhaps we delete this step

Reply:
oh nevermind. The user must specify where gmst and ohc are as inputs to FACTS. The default being the directory listed here. I think this is good to stay. Please disregard my @.